modules/vmss/outputs.tf

```diff
@@ -1,3 +1,4 @@
+# tflint-ignore: terraform_naming_convention # TODO rename to scale_set_name, but bundle with next breaking change
```

Comment:
I simply don't want to pollute the user-facing changelog with a lot of breaking changes to `vmss`. It's far from being usable.